acquisitions/src/main/resources/thunderjet/mod-invoice/invoice.feature

```diff
@@ -3,11 +3,22 @@ Feature: mod-invoice integration tests
   Background:
     * url baseUrl
     * table modules
-      | name                |
-      | 'mod-invoice'       |
-      | 'mod-login'         |
-      | 'mod-permissions'   |
-      | 'mod-configuration' |
+      | name                        |
+      | 'mod-permissions'           |
+      | 'mod-configuration'         |
+      | 'mod-login'                 |
+      | 'mod-users'                 |
+      | 'mod-pubsub'                |
+      | 'mod-orders'                |
+      | 'mod-orders-storage'        |
+      | 'mod-invoice'               |
+      | 'mod-invoice-storage'       |
+      | 'mod-finance'               |
+      | 'mod-finance-storage'       |
+      | 'mod-organizations'         |
+      | 'mod-organizations-storage' |
+      | 'mod-inventory'             |
+      | 'mod-inventory-storage'     |
```

Comment:
Interesting how tests worked previously without these dependencies, or after switch to busybee we have not adopted config file to runt it locally?

Reply:
> Interesting how tests worked previously without these dependencies, or after switch to busybee we have not adopted config file to runt it locally?

The added inventory modules are only required for my tests, previously entire mod-invoice ran fine without these module changes, because Azizbek already fixed it

Reply:
Note: This feature is not being used in InvoiceApiTest. There are two approach to write unit test. First one to use place all tests in invoice.feature that readable generate cucumber report or Only keep setup in invoice-junit.feature and put all tests in InvoiceApiTest class. Currently acq tests choosing second approach.

Reply:
> Note: This file is not being used in InvoiceApiTest. There are two approach to write test. First one to place all tests in invoice.feature that readable generate cucumber report or Only keep setup in invoice-junit.feature and put all tests in InvoiceApiTest class. Currently acq tests choosing second approach.

Thanks, noted, I am rewriting these tests and will include your remarks